instrumental/gui.py

```diff
@@ -354,7 +354,7 @@ def _lut_scale_image(self, arr):
     def _create_lut(self, k):
         A = 2**15
         B = 100  # k's scaling factor
-        g = lambda i, k: A*((k-B)*i) / ((2*k)*x - (k+B)*A)
+        #g = lambda i, k: A*((k-B)*i) / ((2*k)*x - (k+B)*A)
```

Comment:
I want to avoid unnecessary pep8-only changes, so let's revert this.

Reply:
well here, x is just not defined, I tought maybe it should be a parameter of the lambda function. And this method doesn't return anything?

Anyway I reverted it